src/navigation/ViewPagerAdapter.js

```diff
@@ -120,7 +120,7 @@ class ViewPagerBackend extends React.Component {
       transition,
       showPageIndicator,
       pageMargin,
-      overdrag,
+      overdrag = true,
```

Comment:
FYI, there was a breaking change in pager view where overdrag is no longer default to true